guidelines/terms/20/programmatically-determined.html

```diff
@@ -6,8 +6,7 @@
       <a>user agents</a>, including <a>assistive technologies</a>, can extract and present this information to users in different modalities
    </p>
    
-   <aside class="example"><p>Determined in a markup language from elements and attributes that are accessed directly
-      by commonly available assistive technology.
+   <aside class="example"><p>Determined in a markup language from elements and attributes that are exposed in the accessibility tree.
```

Comment:
```suggestion
   <aside class="example"><p>Determined in a markup language from elements and attributes that are accessed via the accessibility tree by commonly available assistive technology.
```
As I mentioned in a [comment](https://github.com/w3c/wcag/issues/3001#issuecomment-1452535712) today, I'd prefer to keep "commonly available assistive technology" in both examples, consistent with [5.2.4](https://www.w3.org/TR/WCAG/#cc4) and "[accessibility supported](https://www.w3.org/TR/WCAG/#dfn-accessibility-supported)."

Reply:
i'd keep the aspect of whether something is in the accessibility tree, and whether or not AT then accesses/uses it for anything, separate

Reply:
I prefer 'exposed', as it is rather about clean html instead of depending on abilities of existing technologies.